app/views/login/confirm.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="container">
+    <% flash.each do |type, msg| %>
+      <div class="<%=type.to_s === "success" ? 'alert alert-success mt-2 mb-4' : 'alert alert-danger mt-2 mb-4'%>">
+        <%= msg %>
+      </div>
+    <% end %>
+  </div>
+  <div class="px-4 py-3 my-5 text-center">
+    <h2 class="display-5 fw-bold pb-1">Welcome!</h2>
+    <h3 class="display-5 pb-2">Please Verify All the information below.</h3>
+      <%= form_with(model: @user, local: true) do |f| %>
+        <div class="row mb-3">
+          <div class="col-md-6 offset-md-3">
+            <div class="col-md-6 offset-md-3">
+              <%= f.label :first_name, class: "form-label" %>
+              <%= f.text_field :first_name, value: @user.first_name, :required => true, class: "form-control" %>
```

Comment:
added :required fields to text inputs on the form to validate on the frontend that fields aren't empty